### DISCUSSION TITLE
multi threaded for write?

### DISCUSSION THREAD

**Discussion by sprappcom** [tarantool/tarantool#10282] [Q&A]
possible to do multi threading for writes?

**Comment by sergos** [comment]
Hi! No, there's a lightweight fiber multitasking that should help you.
Although, it depends on how you want to interact with Tarantool. If you plan to do it via network - it's already there, you can use option of `iproto_threads` and then multiple connections will be served by the designated number of iproto threads, each of which will write in it's own fiber inside the TX thread.
We plan to introduce a dedicated [interface](https://github.com/orgs/tarantool/discussions/10635) for the libraries you want to run along with Tarantool in a separate thread, so stay tuned.
